pycytominer/tests/test_aggregate.py

```diff
@@ -10,7 +10,8 @@
 tmpdir = tempfile.gettempdir()
 
 # Setup a testing file
-test_output_file = os.path.join(tmpdir, "test.csv")
+test_output_file1 = os.path.join(tmpdir, "test.csv")
+test_output_file2 = os.path.join(tmpdir, "test.parquet")
```

Comment:
```suggestion
OUTPUT_FILE_CSV = os.path.join(tmpdir, "test.csv")
OUTPUT_FILE_PARQUET = os.path.join(tmpdir, "test.parquet")
```

More specific than 1 and 2, and easier to parse.

Reply:
Also note the capitalization to keep this test consistent with the other `test_.py` files.